CrewChiefV4/ACS/ACSGameStateMapper.cs

```diff
@@ -413,33 +495,49 @@ public GameStateData mapToGameStateData(Object memoryMappedFileStruct, GameState
                             currentGameState.SessionData.PlayerBestLapSector3Time = currentGameState.SessionData.LastSector3Time;
                         }
                     }
-                    else if (currentGameState.SessionData.SectorNumber == 2)
+                    else if (currentGameState.SessionData.SectorNumber == 2 )
                     {
-                        
+
                         //Console.WriteLine("sector1 time:{0:D2}m:{1:D2}s:{2:D3}ms", ts.Minutes, ts.Seconds, ts.Milliseconds);
                         currentGameState.SessionData.SessionTimesAtEndOfSectors[1] = currentGameState.SessionData.SessionRunningTime;
+
                         // TODO: confirm that an invalid sector will put -1 in here...
+                        if (isOnline)
+                            sectortime = addPlayerLapdata(sectortime, currentGameState.SessionData.SessionRunningTime);
                         currentGameState.SessionData.LastSector1Time = sectortime;
+                        TimeSpan span = TimeSpan.FromSeconds(currentGameState.SessionData.LastSector1Time);
+                        Console.WriteLine("sector1 time:{0:D2}m:{1:D2}s:{2:D3}ms", span.Minutes, span.Seconds, span.Milliseconds);
+
                         if (currentGameState.SessionData.LastSector1Time > 0 &&
                             (currentGameState.SessionData.PlayerBestSector1Time == -1 || currentGameState.SessionData.LastSector1Time < currentGameState.SessionData.PlayerBestSector1Time))
                         {
                             currentGameState.SessionData.PlayerBestSector1Time = currentGameState.SessionData.LastSector1Time;
                         }
                     }
-                    if (currentGameState.SessionData.SectorNumber == 3)
+                    if (currentGameState.SessionData.SectorNumber == 3 )
                     {
-                        
-                        //Console.WriteLine("sector2 time:{0:D2}m:{1:D2}s:{2:D3}ms", ts.Minutes, ts.Seconds, ts.Milliseconds);
+
+
                         currentGameState.SessionData.SessionTimesAtEndOfSectors[2] = currentGameState.SessionData.SessionRunningTime;
+                        if (isOnline)
+                            sectortime = addPlayerLapdata(sectortime, currentGameState.SessionData.SessionRunningTime);
```

Comment:
I know it's a minor thing, but please use curly braces for all if statements, even if it's only a 1-liner:
if (isOnline) 
{
   sectortime = ....
}